e2e/rbd.go

```diff
@@ -1515,6 +1551,43 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+
+			By("create a PVC-PVC clone with object-map,fast-diff image-features and bind it to an app", func() {
+				// pvc clone is only supported from v1.16+
+				if !k8sVersionGreaterEquals(f.ClientSet, 1, 16) {
```

Comment:
This is no longer needed, please see #2617 , in short, the version check below 1.20 is not needed any more, so you can remove this.